runner/runners/invoke.go

```diff
@@ -131,17 +134,16 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 
 	switch st.State {
 	case execer.COMPLETE:
-		srcToDest := map[string]string{
-			stdout.AsFile(): "STDOUT",
-			stderr.AsFile(): "STDERR",
+		tmp, err := temp.NewTempDir("", "invoke")
```

Comment:
This discussion is all correct, but also I think it might be easier to have Invoker also hold a tmp instead of needing to create a new one right here.